qiskit/circuit/controlledgate.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         definition: Optional["QuantumCircuit"] = None,
         ctrl_state: Optional[Union[int, str]] = None,
         base_gate: Optional[Gate] = None,
+        condition: Optional[Tuple] = None,
```

Comment:
The type can be explicitly
```python
condition: Optional[Tuple[Union[ClassicalRegister, Clbit], int]] = None,
```
to get the full type check (and similar in `Gate`).